terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
@yoshikawaa
Yes. I was thinking about making similar modifications for any branch.

If we limit the target to be modified to 5.4.0, it is possible to implement a better implementation corresponding to this issue,
I think that it is not necessary to change the design of this class.

For this reason, I add to JavaDoc that it is dividing methods considering extensibility.